app/assets/javascripts/application.js

```diff
@@ -17,9 +17,48 @@
 //= require_tree .
 $(document).ready(function() {
   chooseCharacter();
-
+  // sellItem();
 });
 
+function sellItem(id){
+    $(id).toggleClass('hide');
+    $(id).toggleClass('show');
+}
+
+function buyItem(id){
+    $(id).toggleClass('hide');
+    $(id).toggleClass('show');
+}
+
+
+function soldItems(){
+    var sold_items = $("table.character_trades").find(".show")
+
+    item_classes = []
+    $.each(sold_items, function(index, sold_item){
+        item_classes.push(this.className)
+
+    })
+    data = {"classes": item_classes}
+    $.post("/trades", data)
+}
+
+function itemExchange(){
+  var sold_items = $("table.character_trades").find(".show")
```

Comment:
Identifier 'sold_items' is not in camel case.<br>Mixed double and single quotes.<br>Missing semicolon.